service/controller/cluster.go

```diff
@@ -109,9 +179,435 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
 		}
 	}
 
-	c := &Cluster{
+	return &Cluster{
 		Controller: operatorkitController,
+	}, nil
+}
+
+func newClusterResources(config ClusterConfig, certsSearcher certs.Interface) ([]resource.Interface, error) {
```

Comment:
resources are defined in this function instead of using the `resource_set.go` file